src/Sarif/SplittingStrategy.cs

```diff
@@ -63,5 +63,7 @@ public enum SplittingStrategy
         /// I.e., the total number of log files created is the sum of the unique fingerprint by organization, entity type, and repository.
         /// </summary>
         PerRunPerOrgPerEntityTypePerRepositoryPerPartialFingerprint,
+
+        PerFingerprint,
```

Comment:
Add a comment:

A grouping strategy that coalesces locations for results with a common fingerprint into a single result. I.e., the total number of results within a run equals the number of unique fingerprints within the run.

Reply:
we won't need this. will remove

---
In reply to: [579716158](https://github.com/microsoft/sarif-sdk/pull/2289#discussion_r579716158) [](ancestors = 579716158)